app/src/main/java/org/schabi/newpipe/MainActivity.java

```diff
@@ -373,6 +387,12 @@ private void showTabs() throws ExtractionException {
         serviceArrow.setImageResource(R.drawable.ic_arrow_down_white);
 
         //Tabs
+
+        //Home button
+        drawerItems.getMenu()
```

Comment:
Shouldn't the menu button be hidden in the MainActivity? There is no need for a "home" button if we are already at home ;-)

Reply:
Why tho? I mean most apps have it I guess, If you want me to take it off I'll do it :)

Reply:
Oh ok, if it's normal to leave it in, then feel free to ignore my request ;-)

Reply:
I agree that the home button should also be visible on the home page.